src/cfnlint/config.py

```diff
@@ -221,6 +221,24 @@ def comma_separated_arg(string):
     return string.split(',')
 
 
+def parse_configure_rules(string):
+    """ Parse the config rule structure """
+    configs = comma_separated_arg(string)
+    results = {}
+    for config in configs:
+        try:
+            rule_id = config.split(':')[0]
+            config_name = config.split(':')[1].split('=')[0]
+            config_value = config.split(':')[1].split('=')[1]
+            if rule_id not in results:
+                results[rule_id] = {}
+            results[rule_id][config_name] = config_value
+        except Exception as e:
+            raise e
```

Comment:
this could/should be a better error message. It'a pretty generic try/catch around an expected format in a configuration